anmykh/homework/homework_6.py

```diff
@@ -0,0 +1,39 @@
+def max_function(*items, key=None):
+    if len(items) == 0:
+        return ValueError
+    if len(items) == 1:
+        items = tuple(items[0])
+    if len(items) == 0:
+        return ValueError
+    max_value = items[0]
+    for item in items[1:]:
+        if item > max_value:
```

Comment:
key never used

use key to compare not items , but values calculated using key function